### ISSUE TITLE
Quotes and FV not in sync

### ISSUE THREAD

**Issue by JamesBura** [ctubio/Krypto-trading-bot#764]
Hello Everyone, 
K has been awesome for me over the last couple months but recently within the last week I've developed a problem with K not getting the correct market data. If I clear the db it will display toe correct market data in the UI and calculate a good FV, after some time it starts to drift until the quotes are far off from the market. I'm using coinbase pro LTC/USD. Previously I've had good luck with the program so I don't know what has happened. 
I was running in Ubuntu in win10 and it worked great before, I tried a fresh install and the issue did not go away. I deleted that I installed Debian hoping it would be different but hasnt helped either.  Ive done make clean diff but that hasnt helped either, I dont know what else it could be at this point. Can anyone recommend some troubleshooting step or ways to debug to figure out my problem please? 

---
```
app exchange: Coinbase/LTC/USD
app version: v0.4.8+57
OS distro: undisclosed
```
![300px-spock_vulcan-salute3](https://cloud.githubusercontent.com/assets/1634027/22077151/4110e73e-ddb3-11e6-9d84-358e9f133d34.png)

**Comment by JamesBura** [comment]
Thank you for replying rezart and ctubio, so far --database=:memory: seems to have corrected the issue for the small sample I have taken with a couple trades. 
--debug-secret doesn't seem to be printing anything in the CLI though. I have tried --debug-orders and quotes previously but the data was too much too fast to make anything of it. I will report back after I let this run for some more time to get a better idea if it holds. 
Thanks again for your assistance.

**Comment by JamesBura** [comment]
Update, its running as it should now in regards to the market quotes and FV calculations. 
I did notice that it will not save trade data, (pings & pongs) if you have to restart. Can that be written to disk or is this just a workaround to get everything working?

**Comment by ctubio** [comment]
i hope this will be fixed together with #769
